src/components/MapComponent.vue

```diff
@@ -0,0 +1,262 @@
+<template>
+  <div id="map-component">
+    <b-button class="label" type="submit" @click="showLabels">
+      {{`${(this.labelsVisible ? "Hide" : "Show")} Block Numbers`}}
+    </b-button>
+  </div>
+</template>
+
+<script>
+import { mapState } from 'vuex';
+import { loadModules } from 'esri-loader';
+import mapConstants from '../constants/mapConstants';
```

Comment:
Moved unchanging objects that do not rely on ArcGIS lazy load to mapConstants to remove some clutter.